docs/platforms/python/migration/1.x-to-2.x.mdx

```diff
@@ -61,26 +55,92 @@ The API function `last_event_id()` was removed. The last event ID is still retur
 
-<SignInNote />
+- The **global scope** holds data that doesn't change as long as your app is running, for instance the current release.
+- The **isolation scope** follows a single basic unit of your program such as a single request-response cycle or one task in a task queue and holds data pertaining to it. In most cases, the lifecycle of an isolation scope maps to the lifecycle of a transaction.
+- The **current scope** wraps a smaller part of the code, often something that's being tracked by a single span.
+
+The SDK will take care of deciding what data belongs on which scope as long as you use the top-level API, so for instance `sentry_sdk.set_tag()` as opposed to `scope.set_tag()`.
+
+The old APIs will continue to work in 2.0, but we recommend migrating to their new counterparts as soon as possible since they will be removed in the next major release.
+
+### Hub Cloning
+
+Creating a new isolation scope provides a similar level of isolation to what you'd previously get by using a hub clone:
+
+```python diff
+- hub = Hub(Hub.current)
+- # do something
++ with isolation_scope() as scope:
++     # do something
+```
+
+### Scope Configuring
+
+In case of `configure_scope()`, you will need to modify either the isolation scope
+or the current scope, depending on whether the change you're making to the scope
+should affect the whole transaction (one request-response cycle, one execution of
+a task, and so on) or just a specific part of the code.
+
+If you want your scope change to affect a smaller unit of code such as a span, use the current scope.
+
+```python diff
+- with configure_scope() as scope:
+-     # do something with scope
++ scope = Scope.get_current_scope()
++ # do something with scope
+```
+
+If you want your scope change to affect the whole transaction, use the isolation scope.
+
+```python diff
+- with configure_scope() as scope:
+-     # do something with scope
++ scope = Scope.get_isolation_scope()
++ # do something with scope
+```
+
+Additionally, you can no longer mutate a manually-created transaction in a `configure_scope` block. Instead, you need to get the current scope to mutate the transaction. Here is a recipe on how to change your code to make it work:
 
 ```python diff
 transaction = sentry_sdk.transaction(...)
 
 # later in the code execution:
 
--with sentry_sdk.configure_scope() as scope:
--    scope.set_transaction_name("new-transaction-name")
+- with sentry_sdk.configure_scope() as scope:
+-     scope.set_transaction_name("new-transaction-name")
+
++ scope = sentry_sdk.Scope.get_current_scope()
++ scope.set_transaction_name("new-transaction-name")
+```
+
+### Scope Pushing
+
+Scope pushing translates to forking a scope in `2.0`. Similarly as above, you either want to fork the current or the isolation scope, depending on how long the new scope should be active. If it should encompass the whole transaction, fork the isolation scope, otherwise fork the current scope.
```

Comment:
```suggestion
Scope pushing translates to forking a scope in `2.0`. You either want to fork the current or the isolation scope, depending on how long the new scope should be active. If it should encompass the whole transaction, fork the isolation scope, otherwise fork the current scope.
```